Pod/Classes/ios/NYTPhotosOverlayView.m

```diff
@@ -49,6 +50,10 @@ - (void)layoutSubviews {
     }];
     
     [super layoutSubviews];
+
+    if ([self.captionView respondsToSelector:@selector(setPreferredMaxLayoutWidth:)]) {
```

Comment:
> Whereas `respondsToSelector:` doesn't lock anything:

`respondsToSelector:` itself doesn't lock anything, but `lookUpImpOrForward` (which is [called by](https://github.com/opensource-apple/objc4/blob/17d47b3989923007fa052f346b05a32fca389465/runtime/objc-runtime-new.mm#L5108) `lookUpImpOrNil`) [does get a runtime lock](https://github.com/opensource-apple/objc4/blob/17d47b3989923007fa052f346b05a32fca389465/runtime/objc-runtime-new.mm#L5019).